specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/consumer/ackknowledge.message.offset/client.rpt

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app0"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .consumer()
+                                   .groupId("client-1")
+                                   .consumerId("consumer-1")
+                                   .timeout(45000)
+                                   .topic("test")
+                                   .partition(0)
+                                   .partition(1)
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:data.ext ${kafka:dataEx()
+                             .typeId(zilla:id("kafka"))
+                             .consumer()
+                                .group()
+                                 .partition(0)
+                                 .consumer()
+                                     .id("consumer-1")
+                                     .partition(0)
+                                     .build()
+                                 .consumer()
+                                     .id("consumer-2")
+                                     .partition(1)
+                                     .build()
+                                 .build()
+                             .build()}
+
+write zilla:data.ext ${kafka:dataEx()
+                             .typeId(zilla:id("kafka"))
+                             .consumer()
+                               .offsetCommit()
+                                 .progress(0, 2)
+
+                             .build()}
+
+write zilla:data.empty
+write flush
+
+read option zilla:ack 0
```

Comment:
My concern here is that `reserved` is `0` when sending the empty `DATA` frame, so `WINDOW` ack does not need to elevate the `initialAck`, making it difficult to tell the difference before or after ack.

Do you think it's an issue?